src/apis/printInBackground.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Print the current document.
```

Comment:
I suggest we explain a little bit about the difference between `printInBackground` and `print`. In a user's perspertive I think it can be confusing for seeing two different APIs with the same description.